README.md

```diff
@@ -764,6 +792,9 @@ internal enum L10n {
   }
 }
 ```
+
+If you look closely, you can see that the `apples.count` is present in the `Localizable.strings` and the `Localizable.stringsdict`. In this case, SwiftGen will use the plural to generate the method signatures, because Foundation behaves the same way when passing a key that is used in both files to a method to receive a localized string.
```

Comment:
```suggestion
If you look closely, you can see that in this example, the `apples.count` key is present both in the `Localizable.strings` and the `Localizable.stringsdict`. In this case, SwiftGen will use the plural to generate the method signatures, because Foundation behaves the same way when passing a key that is used in both files to a method to receive a localized string.
Note that if your key is present in your `.stringsdict`, you don't need to also put it in your `.strings` file too in practice. This was done in this example just to show that SwiftGen doesn't fail even if you do
```

Or maybe another alternative to that comment is to:
 - Remove the key in the `.strings` file in the above example
 - Change the comment to say the opposite, like "Note that if the same key is both present in `.strings` and `.stringsdict`, SwiftGen will only consider the one in `.stringsdict` because that's also how Foundation behaves"

Reply:
I like the idea of the alternative comment, as it doesn't incentivize to put the same key in both files. I'll change it